lib/src/actions/get_fee_history.dart

```diff
@@ -0,0 +1,43 @@
+import 'dart:js_interop';
+
+import 'package:wagmi_flutter_web/src/js/wagmi.js.dart';
+
+/// [Documentation API](https://wagmi.sh/core/api/actions/getFeeHistory)
+
+class GetFeeHistoryParameters {
+  GetFeeHistoryParameters({
+    this.chainId,
+    this.blockCount,
+    this.rewardPercentiles,
+    this.blockNumber,
+    this.blockTag,
+  });
+
+  final int? chainId;
+  final int? blockCount;
+  final List<int>? rewardPercentiles;
```

Comment:
According to JS doc the following properties are mandatory :
 -  `blockCount`
 - `rewardPercentiles`